src/write_hdf5_xml/write_hdf5_xml.c

```diff
@@ -2659,6 +2706,10 @@ int main(int argc, char **argv)
       sprintf(data_source, "ALOS PALSAR");
       sprintf(copyright, "JAXA, METI (%d)", year);
     }
+    else if (strncmp_case(meta->general->sensor, "ERS", 3) == 0) {
+      sprintf(data_source, "%s SAR", meta->general->sensor);
+      sprintf(copyright, "ESA (%d)", year);
+    }
```

Comment:
else
  asfPrintError("Cannot set data source or copyright for %s!", meta->general->sensor)